### DISCUSSION TITLE
I noted that sad talker is running on my cpu, not my gpu

### DISCUSSION THREAD

**Discussion by SEPalmiere** [OpenTalker/SadTalker#750] [Q&A]
Hi. I tried to use sadtalker, but I noted that it is running in my CPU instead of my GPU. I have a RTX 3070. Please how I fix this?

**Comment by SEPalmiere** [comment]
I think it is a problem with confy (automatic1111) webui. I installed sadtalker directly using pyenv and replaced app_sadtalker.py with the correct one, and it is fully working using my GPU. Now I still need to investigate why confy webui is not recognizing mu GPU.

**Reply by weijuncai6** [reply]
i have same problem，investigating.....

**Comment by byteless** [comment]
please post solution.

**Comment by byteless** [comment]
![image](https://github.com/OpenTalker/SadTalker/assets/3484398/db176ead-dcd6-4c23-9616-4bcb5a73fc94)
If you see, face enhancer using mostly CPU and not GPU.
How to enforce that so I can get faster face enhancer?
